src/Symfony/Component/Notifier/Bridge/Firebase/FirebaseTransport.php

```diff
@@ -24,16 +24,20 @@
 
 /**
  * @author Jeroen Spee <https://github.com/Jeroeny>
+ * @author Cesur APAYDIN <https://github.com/cesurapp>
  */
 final class FirebaseTransport extends AbstractTransport
 {
-    protected const HOST = 'fcm.googleapis.com/fcm/send';
+    protected const HOST = 'fcm.googleapis.com/v1/projects/project_id/messages:send';
+
+    private array $credentials;
+
+    public function __construct(#[\SensitiveParameter] array $credentials, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    {
+        $this->credentials = $credentials;
+        $this->client = $client;
```

Comment:
`$this->client` is `protected` in the parent class `AbstractTransport`, so no need to add it here.